install.sh

```diff
@@ -85,55 +159,92 @@ if !([ -f /etc/ros/rosdep/sources.list.d/20-default.list ]); then
 fi
 rosdep update
 
-
-#=====================================#
-# Workspace and Sub Repository Set Up #
-#=====================================#
+instalog "Installing Sub8 ROS dependencies"
+sudo apt-get install -qq binutils-dev
+sudo apt-get install -qq ros-kinetic-ompl
+sudo apt-get install -qq ros-kinetic-spacenav-node
+
+# Get older version of Google protobuf for Gazebo
```

Comment:
The gazebo messages are defaulted to protoc v2.6... The Ubuntu package is v3.0 (https://launchpad.net/ubuntu/+source/protobuf.) The different versions caused mine not to build. (Not sure if this is the case for others)